PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -58,7 +50,7 @@
     ),
 )
 
-protonTablesTask = cms.Task(filteredProtons,protonTable,multiRPTable)
+protonTablesTask = cms.Task(protonTable,multiRPTable)
```

Comment:
(Checking): shouldn't `ctppsProtons` be added to the task in place ot `filteredProtons` here?

Reply:
> (Checking): shouldn't `ctppsProtons` be added to the task in place ot `filteredProtons` here?

@grzanka

Reply:
@grzanka please confirm, but I believe this task should only include `protonTable` and `multiRPTable`. @perrotta, `ctppsProtons` is actually the sequence we would like to strip from the NANOAODification.